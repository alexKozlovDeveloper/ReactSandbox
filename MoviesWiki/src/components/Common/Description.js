import React, { Component } from "react";

const Description = ({ description }) => {
    return  <div>
                {description}
            </div>;
}

export default Description;