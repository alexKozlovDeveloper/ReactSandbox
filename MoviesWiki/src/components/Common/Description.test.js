import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import Description from "./Description";

configure({adapter: new Adapter()});

test('Description rendering test', () => {

    var description = "someDescription";

    const component = shallow(<Description description={description} />);
    expect(shallowToJson(component)).toMatchSnapshot();
 });

