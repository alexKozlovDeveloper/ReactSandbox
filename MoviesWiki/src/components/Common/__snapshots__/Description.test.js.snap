// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description rendering test 1`] = `
<div>
  someDescription
</div>
`;
